src/main/java/picard/vcf/MergeVcfs.java

```diff
@@ -54,14 +54,67 @@
 import java.util.List;
 
 /**
- * Combines multiple VCF files into a single file. Input files must be sorted by their contigs
- * and, within contigs, by start position. Throws IllegalArgumentException if the contig lists
- * are not present in the input files, are not identical or if the sample lists are not the
- * same; this class uses the GATK to merge headers, which may throw exceptions if the headers
- * cannot be merged. See VCFUtils.smartMergeHeaders for details.
- * <p/>
- * An index file is created for the output file by default. Using an output file name with a
- * ".gz" extension will create gzip-compressed output.
+ * Combines multiple variant files into a single variant file.
+ *
+ * <h3>Inputs</h3>
+ *  <ul>
+ *      <li>One or more input file in VCF format (can be gzipped, i.e. ending in ".vcf.gz", or binary compressed, i.e. ending in ".bcf").</li>
+ *      <li>Optionally a sequence dictionary file (typically name ending in .dict) if the input VCF does not contain a
+ *          complete contig list and if the output index is to be created (true by default).</li>
+ *  </ul>
+ *  <p>
+ *  The input variant data must adhere to the following rules:
+ *     <ul>
+ *         <li>If there are samples, those must be the same across all input files.</li>
+ *         <li>Input file headers must be contain compatible declarations for common annotations (INFO, FORMAT fields) and filters.</li>
+ *         <li>Input files variant records must be sorted by their contig and position following the sequence dictionary provided
+ *         or the header contig list.</li>
+ *     </ul>
+ * </p>
+ * <p>
+ *     You can either directly specify the list of files by specifying <code>INPUT</code> multiple times, or provide a list
+ *     in a file with name ending in ".list" to <code>INPUT</code>.
+ * </p>
+ *
+ * <h3>Outputs</h3>
+ * A VCF sorted (i) according to the dictionary and (ii) by coordiante. 
+ * <p>
```

Comment:
This seems to be un-needed in the description. An index should be assumed and the ability to remove it is described in the argument. It also isn't a standard use-case, so I'd argue against having it here.

Reply:
Ok.